jaguar_serializer/README.md

```diff
@@ -0,0 +1,100 @@
+[![Build Status](https://travis-ci.org/Jaguar-dart/jaguar_serializer.svg?branch=master)](https://travis-ci.org/Jaguar-dart/jaguar_serializer)
+
+# jaguar_serializer
+Format agnostic Serializer library that can be used in server and client for JSON, mongodb, postgresql, etc
+
+- [Documentations](https://github.com/Jaguar-dart/jaguar_serializer/wiki)
+
+# Getting Started
+
+## Install
+
+
+```yaml
+#pubspec.yaml
+dependencies:
+  jaguar_serializer: ^1.0.0-dev1
+
+dev_dependencies:
+  build_runner: ^0.8.0
+  jaguar_serializer_cli: ^1.0.0-dev1
+```
+
+## Simple serializer
+
+Create a file for your model.
+
+```dart
+import 'package:jaguar_serializer/jaguar_serializer.dart';
```

Comment:
Is there no requirement for `library example.user;` anymore?
I  suspect we will get an error `Could not find library identifier so a "part of" cannot be built.`

Reply:
no you don't need it anymore

Reply:
It is a Dart 2.0 feature.